tests/product/test_status.py

```diff
@@ -111,7 +111,7 @@ def test_connection_to_worker_lost(self):
             topology, self.cluster.internal_slaves[1])
         self.check_status(status_output, statuses)
 
-    def test_status_port_not_8080(self):
+    def test_status_port_not_28384(self):
```

Comment:
I think this test is no longer necessary.  The point was to make sure 8080 wasn't hardcoded (since at some point there was a bug where it had been somewhere).